### DISCUSSION TITLE
Remove the "Are you sure you want to exit osu!?" dial when using alt+f4.

### DISCUSSION THREAD

**Discussion by Foodenjoyer25** [ppy/osu#27029] [Ideas]
With the stable client, alt f4'ing was a fast way of closing the game, but now the dial breaks the momentum. since I pressed alt+f4 it's natural to believe that I wanted the game to close unlike pressing escape which can be accidental, so the dial is of no use in this case.
![image_2024-02-04_205533772](https://github.com/ppy/osu/assets/112975360/756568e9-f726-4662-93a4-942caca257a3)

**Comment by mixalisnew** [comment]
i actually like this feature. Also you can press alt f4 twice to fully close or press enter after alt f4 and you are good to go

**Comment by OwenCMYK** [comment]
Yeah I think this is a good feature that should be kept. As stated before you can double press it to start the closing animation instantly, and you can even triple press it to skip that closing animation which makes it way faster than stable. I would however be in favor of the animation automatically skipping if you press Alt+F4 in this dialogue box, because I think if somebody is too impatient to even click the "Let me out!" button, then adding an extra animation before closing might just make them annoyed. And for players who who are experienced enough to double press Alt+F4 to quit quickly, it will prevent them from having to triple press it. So that would be good all around I think.
